.github/workflows/release.yml

```
@@ -1,37 +1,16 @@
-name: Deployment
+name: Publish Release
 
 on:
-  release:
-    types: [ published ]
+  workflow_dispatch:
+  repository_dispatch:
+    types: [ stdlib-release-pipeline ]
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-
-      # Setup Ballerina Environment
-      - name: Set Up Ballerina
-        uses: ballerina-platform/setup-ballerina@v1.1.0
-        with:
-          version: 2201.2.1
-
-      # Build Ballerina Project
-      - name: Ballerina Build
-        run: bal pack ./ballerina
-        env:
-          JAVA_HOME: /usr/lib/jvm/default-jvm
-
-      # Build Module Examples
-      - name: Ballerina Examples Build
-        run: chmod +x ./examples/build.sh && ./examples/build.sh build
-        env:
-          JAVA_HOME: /usr/lib/jvm/default-jvm
-
-      # Push to Ballerina Central
-      - name: Ballerina Push
-        run: bal push
-        working-directory: ./ballerina
-        env:
-          BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_DEV_ACCESS_TOKEN }}
-          JAVA_HOME: /usr/lib/jvm/default-jvm
+  call_workflow:
+    name: Run Release Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/release-package-template.yml@main
```

Shall we change this?
```suggestion
    uses: ballerina-platform/ballerina-standard-library/.github/workflows/release-package-connector-template.yml@main
```